azure_batch_pool - support for the `network_configuration` property (#4088) (#5392)
terraform-providers/terraform-provider-azurerm: website/docs/d/batch_pool.html.markdown

* `user_name` - The user name to log into the registry server.

* `password` - The password to log into the registry server.

---

> A `network_configuration` block supports the following:

Reviewer: Could we make the language here consistent with the rest of the data sources?
```suggestion
A `network_configuration` block exports the following:
```

> * `subnet_id` - (Optional) The ARM resource identifier of the virtual network subnet which the compute nodes of the pool will join. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `subnet_id` - (Optional) The ARM resource identifier of the virtual network subnet which the compute nodes of the pool are joined too.
```

> * `endpoint_configuration` - (Optional) A list of inbound NAT pools that can be used to address specific ports on an individual compute node externally. Set as documented in the inbound_nat_pools block below. Changing this forces a new resource to be created.

Reviewer: And so forth
```suggestion
* `endpoint_configuration` - (Optional) The inbound NAT pools that are used to address specific ports on the individual compute node externally.
```
